seatunnel-connectors-v2/connector-http/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/config/HttpParameter.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.http.config;
+
+import org.apache.seatunnel.shade.com.typesafe.config.Config;
+
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Data
+public class HttpParameter implements Serializable {
+    private String url;
+    private String method;
+    private Map<String, String> headers;
+    private Map<String, String> params;
+    private String body;
+    public void buildWithConfig(Config pluginConfig) {
+        // set url
+        this.setUrl(pluginConfig.getString(HttpConfig.URL));
```

Comment:
Please use `pluginConfig.hasPath(HttpConfig.URL)` to judge whether `HttpConfig.URL` in pluginConfig. `pluginConfig.getString(HttpConfig.URL)` will throw `NullPointException` when `HttpConfig.URL` not in pluginConfig.

Reply:
![image](https://user-images.githubusercontent.com/51053924/182314454-7ea91fc8-bf67-492a-a4d0-962b2e66c78a.png)

Because in prepare stage it is alreay check the url is or not existed, so the scenario that you said will not happen. This method can only be executed after a check.

Reply:
> ![image](https://user-images.githubusercontent.com/51053924/182314454-7ea91fc8-bf67-492a-a4d0-962b2e66c78a.png)
> 
> Because in prepare stage it is alreay check the url is or not existed, so the scenario that you said will not happen. This method can only be executed after a check.

That's ok, Thank you .